src/core/classes/TableSchema.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
these changes could be made in another pull request

Reply:
nai tha kanw allo pr :)